README.md

```diff
@@ -1,2 +1,43 @@
-# RustDiscordBot
-Introductory project on using rust making some kind of advanced discord bot
+
+# Project Title
+
+Hello World Initial Project
+
+## Installation
+
+Go to this website to install rust
+https://www.rust-lang.org/tools/install
+
+
+```
+```
+## Contributing
+
+Contributions are always welcome!
+
+See `contributing.md` for ways to get started.
+
+Please adhere to this project's `code of conduct`.
+
+## Running Tests
+
+tbd...
+
+```
+```
+
+## Authors
+
+- [@JarrodCarson](https://github.com/JarrodCarson)
+
+- [[@mamaluigie](https://github.com/mamaluigie)
```

Comment:
Looks like an extra anomalous left bracket

Reply:
this is true